python/lsst/analysis/tools/actions/scalar/scalarActions.py

```diff
@@ -153,6 +153,7 @@ class FracThreshold(ScalarFromVectorAction):
     threshold = Field[float](doc="Threshold to apply.")
     percent = Field[bool](doc="Express result as percentage", default=False)
     relative_to_median = Field[bool](doc="Calculate threshold relative to " "the median?", default=False)
+    absolute_value = Field[bool](doc="Take absolute value of distribution")
```

Comment:
Should this have a default? (i.e., `default=False`?)

Reply:
Yes, but I'm moving it to DM-39256 anyway.